tf2onnx/onnx_opset/math.py

```diff
@@ -399,3 +399,9 @@ def version_7(cls, ctx, node, **kwargs):
         ctx.remove_node(node.name)
         ctx.make_node(op_type="Sub", inputs=[node.input[0], mul.output[0]],
                       name=node.name, outputs=node.output, shapes=shapes, dtypes=dtypes)
+
+@tf_op("Selu")
+class Selu:
+    @classmethod
+    def version_10(cls, ctx, node, **kwargs):
```

Comment:
Selu is there since opset 1:
https://github.com/onnx/onnx/blob/master/docs/Operators.md#selu
want to enable it for all opsets ?

Reply:
Changed to version_4, which seems to be the earliest so far.

Reply:
sold.